### ISSUE TITLE
cargo run shows correct output. Running the built executable does not.

### ISSUE THREAD

**Issue by nikita-skobov** [TimonPost/crossterm#187]
Hi, thanks for the great library. It does exactly what I need, but I noticed an issue on Windows 10 using git bash terminal.

Minimal example:

```rs
use crossterm::{ClearType, Crossterm};
use std::io::{self, stdout, Write};
use std::{thread, time::Duration, time};


pub fn read_sync2() -> io::Result<()> {
  let crossterm = Crossterm::new();
  let terminal = crossterm.terminal();
  let cursor = crossterm.cursor();

  terminal.clear(ClearType::All)?;

  cursor.hide()?;
  cursor.goto(0, 2)?;
  println!("Test: {}", "some data");

  cursor.goto(0, 8)?;
  println!("Test more: {}", "3213214214421");

  cursor.show()?;
  Ok(())
}

fn main() {
  read_sync2();
}

```

`cargo run`

provides the output as expected:

```
Test: some data





Test more: 3213214214421
```

But running `cargo build` and then running the executable directly in the same git bash terminal I get:

```
?[2J?[1;1H?[3;1HTest: some data
?[9;1HTest more: 3213214214421
```

Ive tested this both for debug, and release options with cargo build. I have also tried uninstalling git bash and  reinstalling so that it uses the CMD.exe instead of the seperate terminal emulator, and I get the same results.

**Comment by TimonPost** [comment]
HI! Do you mean that with CMD, you get the correct results? And that you only encounter problems with gitbash?

**Comment by nikita-skobov** [comment]
Closing the issue because I think I figured it out. It turns out I was partly wrong. If using the regular git bash terminal, crossterm always shows the results that include the escape characters: `?[2J?[1` etc.
However, when I made this issue I was using VS code's terminal, and in the terminal when I ran the built program, I got the same results. If instead of running the built program, I did `cargo run` then it shows the results correctly. I noticed at the top of the terminal pane, VS code switches the terminal emulator on my behalf whenever using `cargo run`. So crossterm still does not work with git bash (and I know this is a seperate issue) and my issue was more with VS code than with this project.

so I will be closing this issue, and sorry for the late response.
